oauth2server/apps/accounts/tests/test_user_admin.py

```diff
@@ -1,34 +1,36 @@
 import base64
+import pdb
 
 from rest_framework.test import APIClient
 from rest_framework import status
 from django.test import TestCase
 
+
 class UserAdministrationTest(TestCase):
 
     fixtures = ['test_credentials',]
 
     def setUp(self):
         self.api_client = APIClient()
 
-    # Make sure that client_id with admin privilege can't update the account if it's in a locked state.
+    # Make sure that client_id with admin privilege can update the account if it's in a locked state.
     # Use john3@doe.com which has a locked account
-    def test_locked_account(self):
+    def test_unlock_locked_account(self):
```

Comment:
This cover just the positive test. You stil miss : 
1.          except (IntegrityError, InternalError, DataError, DatabaseError): 
2.         if not request.user.account_is_verified: